twilio.js

```diff
@@ -49,18 +52,22 @@ Twilio.prototype.sendSMS = function (options, callback) {
  * {String} options.body
  * {String} options.mediaUrl
```

Comment:
Not something you did --- but this should be `[options.mediaUrl]` since it is optional